packages/components/src/LightBox/LightBox.module.css

```diff
@@ -121,16 +118,81 @@
   max-width: 600px;
   max-height: 90px;
   margin: 0 auto var(--space-larger);
+  padding: 0 var(--space-base);
   overflow-y: auto;
-  text-align: center;
   flex-direction: column;
   flex-shrink: 0;
   gap: var(--space-small);
 }
 
+.title {
+  text-align: center;
+}
+
 .blurOverlay {
   position: fixed;
   inset: 0;
   backdrop-filter: blur(30px);
   background-color: rgba(3, 43, 58, 0.25);
 }
+
+.thumbnailBar {
+  display: flex;
+  position: relative;
+  z-index: var(--elevation-base);
+  width: 100%;
+  min-height: 90px;
+  margin-bottom: var(--space-small);
+  padding-bottom: var(--space-small);
+  overflow-x: scroll;
+  justify-content: safe center;
+  align-items: center;
+  gap: var(--space-small);
+  flex-shrink: 0;
+}
+
+.thumbnailImage {
+  width: 100%;
+  height: 100%;
+  border-radius: var(--radius-small);
+  opacity: 0.8;
+  object-fit: cover;
+}
+
+.thumbnail {
+  width: var(--space-extravagant);
+  height: var(--space-extravagant);
+  border-radius: var(--radius-small);
+  background-color: var(--color-text--reverse);
+  transition:
+    border var(--timing-base) ease-in,
+    transform var(--timing-quick) ease-in-out;
+  flex-shrink: 0;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.thumbnail.selected {
+  margin: 0 var(--space-smaller);
+  padding: var(--space-smallest);
+  border: 2px solid var(--color-text--reverse);
```

Comment:
similar thought here. these are the first two times we'd be using the text color for non text. I'm hesitant to do it without a compelling reason.